# Ocean Current Message
std_msgs/Header header

# Current velocity
geometry_msgs/Vector3 velocity
